backend/src/app.controller.ts

```diff
@@ -36,4 +38,10 @@ export class AppController {
   getUser(@Req() ID: string): Promise<User | null> {
     return this.userService.getUser(ID);
   }
+
+  // tag/search/?q=
```

Comment:
Is there a reason why this request is inside the normal app controller? Didnt we use single controllers for subroutes like tags?